src/PeopleSDKAdapter.js

```diff
@@ -184,4 +184,26 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
 
     return this.getPersonObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits a list of people that match the given query.
+   * An empty array is returned when there are no matches.
+   *
+   * @param {string} query Search query
+   * @returns {external:Observable.<Person[]>} Observable that emits person list based on search query
+   */
+  searchPeople(query) {
+    return defer(() => this.datasource.people.list({displayName: query}))
+      .pipe(
+        catchError((error) => {
+          logger.error('PEOPLE', undefined, 'searchPeople()', 'error fetching person list', error);
+          throw error;
+        }),
+        map((peopleList) => {
+          logger.debug('PEOPLE', undefined, 'searchPeople()', ['emit persons list', peopleList.items]);
+
+          return peopleList.items;
```

Comment:
Is the format of this list an array of [People objects](https://github.com/webex/component-adapter-interfaces/blob/master/src/PeopleAdapter.js#L6)?

Reply:
Yes it contains the list of all the people matching the search query in an array.

Reply:
@harsh789rai there might a difference between a people object as returned by the SDK and a people object as defined in the adapter.

Check if the objects in `peopleList.items` respect the adapter format described in the link Lalli provided https://github.com/webex/component-adapter-interfaces/blob/master/src/PeopleAdapter.js#L6

If not, you should map from sdk format to adapter format, see for example the `fromSDKActivity` function in the Activities adapter which does the same for activities.

Reply:
I have created a new private function fromSDKPeople to map sdk data to adapter data